### PR TITLE
Adding test for prepare_url, also added code of conduct and issue template

### PR DESCRIPTION
**Pull request by Kerl1310**
Adding an additional test as requested in #1

### CODE REVIEW

**Comment by jacobtomlinson** [comment]
Thanks for this @Kerl1310!

Would you mind splitting this into two PRs, one for the docs changes and one for the tests?

**Comment by Kerl1310** [comment]
No problem, yeah that's fine, shall do that now

File: tests/test_generators.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import unittest
+try:
+    import unittest.mock as mock
+except ImportError:
+    import mock
```
Comment (jacobtomlinson) [inline tests/test_generators.py]: As you are not mocking anything here you can drop the imports. Once you do this the CI will pass.

**Comment by Kerl1310** [comment]
Removed Code of Conduct and Issue Template from this PR, shall submit in another PR shortly

**Comment by Kerl1310** [comment]
Dropped the imports

**Comment by jacobtomlinson** [comment]
🎉

We love to thank contributors by sending them [opsdroid stickers](https://medium.com/opsdroid/contributor-sticker-packs-738058ceda59)! Fill in the [claim form](https://forms.gle/iQZV9DjkaUiSaDXK6) to get yours!

**Pull request merged**